src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                </tr>
+                <tr>
+                    <td>TOPIC_FUELTANK2_UNITS</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_FUELTANK1_PERCENT</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_FUELTANK2_PERCENT</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_STATE_OF_CHARGE</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_ENGINE_ROAD_SPEED</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_VEHICLE_ACTIVE</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_DRIVER_SEATBELT</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_LEFT_TURN_SIGNAL</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_RIGHT_TURN_SIGNAL</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_EV_CHARGING_STATE</td>
+                </tr>
+                <tr>
+                    <td>TOPIC_PARK_BRAKE</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Buzzer Beep (0x24)</h3>
+        <p>Sent from an IOX to the GO device to request the buzzer beep with the given parameters.</p>
+        <h4>Payload</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0</td>
+                    <td>Number of Beeps</td>
+                </tr>
+                <tr>
+                    <td>1</td>
+                    <td>Duration (Multiple of 56ms)</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Delay Between Beeps (Multiple of 56ms)</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>IOX Request/Status (0x25)</h3>
+        <p>Sent from the IOX to the GO device to inform the GO device of events or status changes.</p>
+        <h4>Payload</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>Information Type</td>
+                </tr>
+                <tr>
+                    <td>2-7</td>
+                    <td>Optional bytes that depend on the type</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 0 - Busy</h3>
+        <p>This message indicates to the GO device that the issuing IOX is busy with a critical task and that the GO should not enter the sleep state. The IOX should send this message again to release the GO device once it has completed its critical tasks.</p>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on